csharp-basic-topics/TaskCompletedTaskFromResultVsReturnInCSharp/Tests/TaskCompletedVsTaskFromResultVsReturnTests.cs

```diff
@@ -0,0 +1,35 @@
+namespace TaskCompletedVsTaskFromResultVsReturnInCSharpTests;
+
+public class TaskCompletedVsTaskFromResultVsReturnTests
+{
+    [Fact]
+    public void WhenCallingUseTaskCompletedAsync_ThenUseTaskCompleted()
+    {
+        var taskCompletedClass = new TaskCompletedHandler();
+
+        var result = taskCompletedClass.UseTaskCompletedAsync();
+
+        Assert.True(result.IsCompletedSuccessfully);
+    }
+
+    [Fact]
+    public async Task WhenCallingUseTaskFromResultAsync_ThenUseFromResult()
+    {
+        var taskFromResultClass = new TaskFromResultHandler();
+
+        var result =  taskFromResultClass.UseTaskFromResultAsync();
+        var message = await result;
```

Comment:
Why don't just do `var message = await taskFromResultClass.UseTaskFromResultAsync()` in a single line?